lib/internal/test_runner/utils.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const { basename } = require('path');
+const kTestFilePattern = /((^test(-.+)?)|(.+[.\-_]test))\.m?js$/;
+
+function doesPathMatchFilter(p) {
+  return kTestFilePattern.test(basename(p));
```

Comment:
```suggestion
  return RegExpPrototypeTest(kTestFilePattern, basename(p));
```

Reply:
Calling `%RegExp.prototype.test%` on `RegExp` object looks up the `exec` property of that object.
```suggestion
  return RegExpPrototypeExec(kTestFilePattern, basename(p)) !== null;
```

Reply:
whoops, thanks, i've got a few packages to update :-)